src/core/dbus-manager.c

```diff
@@ -2598,6 +2620,7 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_METHOD("Subscribe", NULL, NULL, method_subscribe, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("Unsubscribe", NULL, NULL, method_unsubscribe, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("Dump", NULL, "s", method_dump, SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD("Dump2", NULL, "h", method_dump2, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
Hmm, I don't hink "Dump2" is a great name. Maybe call this "DumpByFileDescriptor" or something a tiny bit more descriptive?